fluentd/Dockerfile.centos7

```diff
@@ -35,11 +35,11 @@ RUN mkdir -p ${HOME} && mkdir -p /etc/fluent/plugin && \
     gem install -N --conservative --minimal-deps --no-ri --no-doc \
      'activesupport:<5' \
      'public_suffix:<3' \
-     'elasticsearch-transport:<5' \
-     'elasticsearch-api:<5' \
-     'elasticsearch:<5' \
+     'elasticsearch-transport:<6' \
+     'elasticsearch-api:<6' \
+     'elasticsearch:<6' \
       fluentd:${FLUENTD_VERSION} \
-     'fluent-plugin-elasticsearch:~>1.0' \
+     'fluent-plugin-elasticsearch:<2.0.0' \
```

Comment:
shouldn't this be `fluent-plugin-elasticsearch:~>2.0` ?

Reply:
According to the readme of fluent-plugin-elasticsearch [1], fluent-plugin-elasticsearch has to be older than 2.0.0 for fluentd v0.12?
```
fluent-plugin-elasticsearch | fluentd      | ruby
        >= 2.0.0            | >= v0.14.20  | >= 2.1
        < 2.0.0             | >= v0.12.0   | >= 1.9
```
[1] https://github.com/uken/fluent-plugin-elasticsearch/

Reply:
ok